server/app.js

```diff
@@ -5,6 +5,7 @@ const cookieParser = require('cookie-parser');
 
 const router = require('./router');
 const dataBase = require('./database/dbConnection');
+const errorHandler = require('./controllers/errors');
```

Comment:
its already added to router  index file ,so delete it from here .& return them to the Router file.